packages/infra/lib/shared/rds.ts

```diff
@@ -53,32 +62,41 @@ export function addDBClusterPerformanceAlarms(
   createAlarm({
     metric: dbCluster.metricCPUUtilization(),
     name: "CPUUtilizationAlarm",
-    threshold: thresholds.cpuUtilizationPct,
+    threshold: dbConfig.alarmThresholds.cpuUtilizationPct,
     evaluationPeriods: 1,
     treatMissingData: cloudwatch.TreatMissingData.NOT_BREACHING,
   });
 
   createAlarm({
     metric: dbCluster.metricVolumeReadIOPs(),
     name: "VolumeReadIOPsAlarm",
-    threshold: thresholds.volumeReadIops,
+    threshold: dbConfig.alarmThresholds.volumeReadIops,
     evaluationPeriods: 1,
     treatMissingData: cloudwatch.TreatMissingData.NOT_BREACHING,
   });
 
   createAlarm({
     metric: dbCluster.metricVolumeWriteIOPs(),
     name: "VolumeWriteIOPsAlarm",
-    threshold: thresholds.volumeWriteIops,
+    threshold: dbConfig.alarmThresholds.volumeWriteIops,
     evaluationPeriods: 1,
     treatMissingData: cloudwatch.TreatMissingData.NOT_BREACHING,
   });
 
   createAlarm({
     metric: dbCluster.metricACUUtilization(),
     name: "ACUUtilizationAlarm",
-    threshold: thresholds.acuUtilizationPct,
+    threshold: dbConfig.alarmThresholds.acuUtilizationPct,
+    evaluationPeriods: 1,
+    treatMissingData: cloudwatch.TreatMissingData.NOT_BREACHING,
+  });
+
+  createAlarm({
+    metric: dbCluster.metricDatabaseConnections(),
+    name: "DatabaseConnectionsAlarm",
+    threshold: 0.8 * getMaxPostgresConnections(dbConfig.maxCapacity),
     evaluationPeriods: 1,
```

Comment:
Maybe move this to two evaluation periods, as well as the ACU utilization? It seems like one is too sensitive?

NABD if we're not comfortable changing the ACU one now.